src/main/kotlin/blackjack/domain/Players.kt

```diff
@@ -0,0 +1,14 @@
+package blackjack.domain
+
+class Players(val value: List<Player>) {
+
+    init {
+        require(value.size >= MIN_NUMBER_PLAYERS) { "우효하지 않는 플레이어 인원 수 입니다. 최소 2명 이상 입력해주세요." }
+        require(value.size <= MAX_NUMBER_PLAYERS) { "우효하지 않는 플레이어 인원 수 입니다. 최대 8명 까지 가능합니다." }
```

Comment:
정말 사소하지만 오타가 있습니다 ㅎㅎ 그리고 선언해두신 상수를 활용한다면 더욱 변경에 대처하기 유연한 구조가 될 것 같아요.
```suggestion
	require(value.size >= MIN_NUMBER_PLAYERS) { "유효하지 않은 플레이어 인원 수 입니다. 최소 ${MIN_NUMBER_PLAYERS}명 이상 입력해주세요." }
        require(value.size <= MAX_NUMBER_PLAYERS) { "유효하지 않은 플레이어 인원 수 입니다. 최대 ${MAX_NUMBER_PLAYERS}명 까지 가능합니다." }
```